workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/Apps/AppCreationFlow/InformLicensingTermsPage/InformLicensingTermsPricePage.tsx

```diff
@@ -168,37 +171,44 @@ export function InformLicensingTermsPricePage({
 				/>
 			</Section>
 
-			<Section
-				label="Developer License prices"
-				tooltip="Developer licenses are limited to 5 unique addresses and should not be used for full scale production deployments."
-				tooltipText="More Info"
-			>
-				{appLicensePrice.developer.length ? (
-					<LicensePriceCard
-						licensePrices={appLicensePrice.developer}
-						onAdd={() => handleAddPriceTier(LicenseTier.DEVELOPER)}
-						onChange={(index: number, price: LicensePrice) =>
-							handleEditPriceTier(
-								LicenseTier.DEVELOPER,
-								index,
-								price
-							)
-						}
-						onDelete={(key: number) =>
-							handleDeletePriceTier(LicenseTier.DEVELOPER, key)
-						}
-					/>
-				) : (
-					<IconButton
-						className="icon-button py-3 w-100"
-						onClick={() =>
-							handleAddPriceTier(LicenseTier.DEVELOPER)
-						}
-					>
-						Add Developer Licenses
-					</IconButton>
-				)}
-			</Section>
+			{isDXP && (
+				<Section
+					label="Developer License prices"
```

Comment:
Could you use translate.
```suggestion
					label={i18n.translate("developer-license-prices")}
```

Reply:
thx!